x/launch/client/cli/tx_create_chain.go

```diff
@@ -54,6 +55,12 @@ func CmdCreateChain() *cobra.Command {
 				}
 			}
 
+			// Read metadata bytes
+			metadataBytes, err := base64.StdEncoding.DecodeString(args[3])
```

Comment:
We should not use any encoding since it just represents unopinionated bytes, we can just convert it with `[]byte(...)`